index.html

```diff
@@ -106,6 +107,7 @@ <h4 class="project-desc">Multi-Post Stories<br /> Gain + Glory</h4>
                             </li>
                         </ul>
                         <button class="btn btn-projects" type="button">See Projects</button>
```

Comment:
- [ ] Please let's make sure that your design is as close to that of Figma. The close button (**X**) of your implementation is not the same as that in Figma in the mobile version. Please let's fix it.
                  
 Your Implementation | Figma Design
   :---:|:---:
  ![](https://user-images.githubusercontent.com/63087491/152258120-c82d23aa-76fa-4ba4-8db2-357b3c4d2faf.png) | ![](https://user-images.githubusercontent.com/63087491/152258130-98d68481-da1d-4f4f-a216-c5c8b34b7de2.png)

- Also, the pop-up for the desktop version of your Design is not pretty much the same as that of Figma. Please let's try to fix it. 

Your Implementation | Figma Design
   :---:|:---:
  ![](https://user-images.githubusercontent.com/63087491/152259656-ddd105b3-db6b-439f-9306-e36bcc491571.png) | ![](https://user-images.githubusercontent.com/63087491/152259660-bb47da60-1aac-47d0-bbf6-ce99e7af7228.png)

Reply:
Hi @FrederickMih 

I have worked on the close button and the desktop version.

thanks for your review